Build errors after updating to 2.9

# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am running the latest version
- [x] I checked the README and found no answer 
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository as it is related to this repository and not to the [official pjsip repository](https://www.pjsip.org)

# Expected Behavior

App to build normally after updating to version 2.9

# Current Behavior

I get build errors

# Failure Information (for bugs)

> Undefined symbols for architecture arm64:
>   "_opus_decode", referenced from:
>       _codec_decode in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>       _codec_recover in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_repacketizer_out", referenced from:
>       _codec_encode in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_repacketizer_get_nb_frames", referenced from:
>       _codec_parse in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>       _codec_encode in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_encode", referenced from:
>       _codec_encode in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_encoder_get_size", referenced from:
>       _codec_open in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_decoder_get_size", referenced from:
>       _codec_open in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_repacketizer_out_range", referenced from:
>       _codec_parse in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_repacketizer_get_size", referenced from:
>       _codec_open in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_repacketizer_cat", referenced from:
>       _codec_parse in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>       _codec_encode in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_encoder_ctl", referenced from:
>       _codec_open in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>       _codec_modify in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_encoder_init", referenced from:
>       _codec_open in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_decoder_init", referenced from:
>       _codec_open in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_packet_get_nb_samples", referenced from:
>       _codec_parse in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>   "_opus_repacketizer_init", referenced from:
>       _codec_open in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>       _codec_parse in libpjmedia-codec-apple-darwin_ios.a(opus.o)
>       _codec_encode in libpjmedia-codec-apple-darwin_ios.a(opus.o)
> ld: symbol(s) not found for architecture arm64


## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Update project with v2.8 to 2.9
2. Try to build

## Context

* Xcode Version: 10.3
* Platform and architecture building for: iOS 9.0 and higher, arm64 arm64e armv7 armv7s
* Operating System: Mac OS 10.14.6

--- a/pjsip.podspec
+++ b/pjsip.podspec
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name         = "pjsip"
-  s.version      = "2.9.0"
-  s.summary      = "pjsip for macOS (OSX)"
+  s.version      = "2.9.0.2"
+  s.summary      = "pjsip for iOS and macOS (OSX)"
   s.homepage     = "http://www.pjsip.org"
   s.author       = 'www.pjsip.org'
   s.source       = { :git => "https://github.com/chebur/pjsip.git", :tag => "#{s.version}" }
@@ -43,7 +43,7 @@ Pod::Spec.new do |s|
   s.ios.vendored_libraries  = [
                                 # 'build/openh264/lib/*.a',
                                 'build/openssl/lib/ios/*.a',
-                                'build/opus/dependencies/lib/*ios.a',
+                                'build/opus/dependencies/lib/*.a',
                                 'build/pjproject/src/pjlib/lib/*ios.a',
                                 'build/pjproject/src/pjlib-util/lib/*ios.a',
                                 'build/pjproject/src/pjmedia/lib/*ios.a',
@@ -73,7 +73,7 @@ Pod::Spec.new do |s|
   s.osx.vendored_libraries  = [
                                 # 'build/openh264/lib/*.a',
                                 'build/openssl/lib/macos/*.a',
-                                'build/opus/dependencies/lib/*darwin18.6.0.a',
+                                'build/opus/dependencies/lib/*.a',
                                 'build/pjproject/src/pjlib/lib/*darwin18.6.0.a',
                                 'build/pjproject/src/pjlib-util/lib/*darwin18.6.0.a',
                                 'build/pjproject/src/pjmedia/lib/*darwin18.6.0.a',

